pkgs/top-level/python-packages.nix

```diff
@@ -6999,6 +6999,10 @@ in {
     withMultimedia = true;
   };
 
+  pyqt5_with_qtlocation = self.pyqt5.override {
+    withLocation = true;
+  };
```

Comment:
We shouldn't introduce multiple variants of pyqt5 because it will create strange bugs when a python environment contains multiple variants of the same package. This override should be done in the package outside of pythonPackages.